js/scriptContacto.js

```diff
@@ -0,0 +1,24 @@
+(function() {
+    
+    (function titleSize(){
+        let windowWidthO = window.outerWidth;
+        let windowWidthI = window.innerWidth;
+        let titulo = document.getElementById("titulo");
+        let fontSize = window.getComputedStyle(titulo).getPropertyValue('font-size');
+        
+        console.log(windowWidthO);
+        console.log(windowWidthI);
+        
+        if(windowWidthO < 1000){
+            let newSize = windowWidthO / 2;
+            newSize += "%";
+            titulo.style.fontSize = newSize;
+        }
+        else if(windowWidthI < 1000){
+            let newSize = windowWidthI / 2;
```

Comment:
Codacy has a fix for the issue: [Replace `············let·newSize·=·windowWidthI·/·2;␍` with `······let·newSize·=·windowWidthI·/·2;`](https://app.codacy.com/gh/Jeffer48/Consultarq/pullRequest?prid=9456239)
```suggestion
      let newSize = windowWidthI / 2;
```